lms/djangoapps/ccx/utils.py

```diff
@@ -292,6 +302,21 @@ def ccx_students_enrolling_center(action, identifiers, email_students, course_ke
                 break
 
             enroll_email(course_key, email, auto_enroll=True, email_students=email_students, email_params=email_params)
+
+        if is_course_licensing_enable and num_identifiers > 1:
```

Comment:
Following the previous comment, what do you think about leaving the is_course_licensing_enable validation and moving the rest of the logic to the plugin?